src/dex-ui-components/base/Notification/Notification.tsx

```diff
@@ -0,0 +1,75 @@
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { Tag, Text, Flex, Link, TagCloseButton, Box } from "@chakra-ui/react";
+import { useCallback } from "react";
+
+export enum NotficationTypes {
+  SUCCESS = "success",
+  WARNING = "warning",
+  ERROR = "error",
+}
+
+interface NotificationProps {
+  type: NotficationTypes;
+  textStyle?: string;
+  message: string;
+  isLinkShown?: boolean;
+  linkText?: string;
+  linkRef?: string;
+  isCloseButtonShown?: boolean;
+  handleClickClose?: () => void;
+}
+
+/**
+ * A component used to communicate to a user a state that affects a system, feature or page.
+ * TODO: Add TSDocs
+ */
+export const Notification = (props: NotificationProps) => {
```

Comment:
A shared component used to display alerts/notifications to the user (e.g. on success of a transaction).